### PR TITLE
fix: support extensionless main field

### LINKED ISSUE #106: Handles packages with no file extension in the main field
It is recommended to leave the extension off in the `package.json/main` field and let NodeJS resolves the right file whether it is `index.js`, `index.node`, `index.cjs`, or `index.mjs`

`size-limit` does not work with this format.

It uses `globby` instead of `NodeJS native resolution here:
https://github.com/ai/size-limit/blob/0a7ae6edaa8ab461c09dff099e09a2ce792679ff/cli-body.js#L339-L344

### PR DESCRIPTION
**Pull request by unional**
closes #106

Added `yarn watch` to run `jest` in watch mode

### CODE REVIEW

**Comment by unional** [comment]
One more checkin. Adding test

File: package.json [hunk 1]
```diff
@@ -64,7 +64,8 @@
   "scripts": {
     "spell": "yaspeller-ci *.md",
     "lint": "eslint-ci *.js test/{fixtures/**/,}*.js",
-    "test": "jest -i && yarn run lint && yarn run spell"
+    "test": "jest -i && yarn run lint && yarn run spell",
+    "watch": "jest --watch"
```
Comment (ai) [inline package.json]: You can run it with `npx jest --watch`

Comment (unional) [reply package.json]: Sure. :) I can remove it.

Comment (ai) [reply package.json]: Yeap, can you remove it?

Comment (unional) [reply package.json]: Removed. 🌷

File: cli-body.js [hunk 2]
```diff
@@ -481,6 +480,14 @@ async function main () {
   return files
 }
 
+function getPackageMain (cwd, mainFile = 'index.js') {
+  try {
+    return [path.relative(cwd, require.resolve(path.join(cwd, mainFile)))]
+  } catch (e) {
```
Comment (ai) [inline cli-body.js]: What kind of error can we get here?

Comment (unional) [reply cli-body.js]: When the file doesn't exist, it throws an cannot find module error:

```sh
Error: Cannot find module '/Users/hwong/code/unional/size-limit/test/fixtures/webpack-multipe-entry-points/index.js'
        at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
        at Function.resolve (internal/modules/cjs/helpers.js:30:19)
        at getPackageMain (/Users/hwong/code/unional/size-limit/cli-body.js:485:40)
        at Promise.all.configFile.config.map (/Users/hwong/code/unional/size-limit/cli-body.js:344:17)
        at Array.map (<anonymous>)
        at main (/Users/hwong/code/unional/size-limit/cli-body.js:335:54)
```

It is observed in the `webpack-multile-entry-points` test.

**Comment by ai** [comment]
Released in 1.3.2

**Comment by unional** [comment]
Thanks!

**Pull request merged**